collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -617,6 +617,7 @@ private void loadForm() {
             }
 
             // Not a restart from a screen orientation change (or other).
+            Timber.w("Setting formController to null");
             Collect.getInstance().setFormController(null);
```

Comment:
I'm wondering if it wouldn't be better to add logs in that `setFormController` method to see the sequence of calls...
Maybe we first set it properly but then to null?

Reply:
Yeah that sounds good! Feel to make that change